src/chainlit/frontend/src/pages/Env.tsx

```diff
@@ -51,18 +45,19 @@ export default function Env() {
   }
 
   const renderInput = (key: string) => {
+    const hasError = !!formik.errors[key];
+
     return (
-      <Box key={key} my={2}>
-        <InputLabel>{key}</InputLabel>
-        <TextField
-          fullWidth
-          className={key}
-          variant="outlined"
-          value={formik.values[key]}
-          error={!!formik.errors[key]}
-          onChange={(e) => formik.setFieldValue(key, e.target.value)}
-        />
-      </Box>
+      <TextInput
```

Comment:
The input seems taller than in the figma 
<img width="1008" alt="Screenshot 2023-07-13 at 17 24 55" src="https://github.com/Chainlit/chainlit/assets/13104895/81fe4841-b8a2-4d26-a951-4ffaca5befdd">